src/projects/create/components/ProjectOutline.js

```
@@ -21,7 +22,7 @@ function ProjectOutline({ project }) {
   return (
     <div className="ProjectOutline">
       <h5>Project Outline</h5>
-      <div className="project-name">{ project.name }</div>
+      <ProjectCardHeader project={project} />
       <div className="project-product">{ productName }</div>
```

Do we need this block now?

---

Do you mean I should remove the header?

---

I guess we do need it, there was no request to remove project name from the outline panel. I'll merge this but please raise the issue if I'm missing something. @vikasrohit

---

@gondzo I was thinking about `productName`. I would check once all changes on in dev.